ui/src/app/applications/components/application-node-info/application-node-info.tsx

```diff
@@ -84,6 +87,18 @@ export const ApplicationNodeInfo = (props: {
                 attributes.push({title: 'HEALTH DETAILS', value: props.controlled.summary.health.message});
             }
         }
+    } else if (props.node && (props.node as ResourceTreeNode).health) {
```

Comment:
```suggestion
    } else if (props.node) {
```
maybe we can write it as such, as line 92 now checks it